scripts/tag_west_repos.sh

```diff
@@ -36,20 +36,27 @@
 
 declare -A PROJECT_TAGS
 
-# Uncomment and edit these lines as needed.
+# Set the tag names for all the projects below.
 #
 #     - keys are project names in west.yml
 #     - values are the tags you want to make
 #
-# PROJECT_TAGS[zephyr]=v2.7.99-ncs1-rc1
-# PROJECT_TAGS[mcuboot]=v1.8.99-ncs1-rc1
-# PROJECT_TAGS[trusted-firmware-m]=v1.5.0-ncs1-rc1
-# PROJECT_TAGS[matter]=v1.9.0-rc1
-# PROJECT_TAGS[openthread]=v1.9.0-rc1
-# PROJECT_TAGS[nrfxlib]=v1.9.0-rc1
-# PROJECT_TAGS[find-my]=v1.9.0-rc1
-# PROJECT_TAGS[nrf-802154]=v1.9.0-rc1
-# PROJECT_TAGS[homekit]=v1.9.0-rc1
+# nrfx/nRF/srcx repositories: vX.Y.Z(-rcN)
+PROJECT_TAGS[nrfxlib]=""
+PROJECT_TAGS[find-my]=""
+PROJECT_TAGS[homekit]=""
+PROJECT_TAGS[matter]=""
+PROJECT_TAGS[nrf-802154]=""
+
+# OSS repositories: vX.Y.Z-ncsN(-rcM)(-I)
+PROJECT_TAGS[zephyr]=""
+PROJECT_TAGS[mcuboot]=""
+PROJECT_TAGS[trusted-firmware-m]=""
+PROJECT_TAGS[tfm-mcuboot]=""
```

Comment:
What about `tfm-mcuboot`, it seems to be the same repository as mcuboot (sdk-mcuboot), just frozen at certain revision - do wee need to tag it seprarately?